package.json

```diff
@@ -16,18 +16,20 @@
     "test-watch": "jest --watch --config ./jest.json"
   },
   "devDependencies": {
-    "@babel/cli": "^7.2.3",
-    "@babel/core": "^7.2.2",
+    "@babel/cli": "^7.4.3",
+    "@babel/core": "^7.4.3",
     "@babel/plugin-transform-modules-umd": "^7.2.0",
-    "@babel/preset-env": "^7.2.3",
+    "@babel/preset-env": "^7.4.3",
     "babel-core": "7.0.0-bridge.0",
-    "babel-jest": "^23.6.0",
-    "browserslist-config-canopy": "^2.0.0",
+    "babel-jest": "^24.7.1",
+    "browserslist-config-canopy": "^3.1.0",
     "in-publish": "^2.0.0",
-    "jest": "^23.6.0",
-    "rimraf": "^2.6.2",
+    "jest": "^24.7.1",
+    "rimraf": "^2.6.3"
+  },
+  "dependencies": {
```

Comment:
shifting `rx` and `rxjs` to `dependencies` reflects the reality that these two libraries are in fact required at runtime. There are also several alternatives to this, including using SystemJS to check/load these dependencies at runtime. Another would be to add them in a `peerDependencies` block, though this option doesn't seem like a good one.

Reply:
I also have a change set queued up that removes support for rx4 altogether. Welcome thoughts on that as well.

Reply:
why is `peerDependencies` not a good option?

Reply:
I'm curious about that also, does `peerDependencies` not work?

Reply:
yes, with a caveat that those dependencies would need to be included in devDependencies also. Happy to experiment with that a bit